defaults/Job.dhall

```diff
@@ -21,4 +22,5 @@ in  { name = None Text
     , services = None (List { mapKey : Text, mapValue : Service })
     , container = None Container
     , concurrency = None Concurrency
+    , permissions = None (List { mapKey : Text, mapValue : Text })
```

Comment:
Here we could have a permission type with all the possibilities and an enum type to represent the values `read|write|none`